aim/sdk/adapters/pytorch_lightning.py

```diff
@@ -88,7 +93,6 @@ def experiment(self) -> Run:
                     log_system_params=self._log_system_params,
                     capture_terminal_logs=self._capture_terminal_logs,
                 )
-                self._run_hash = self._run.hash
```

Comment:
@constd just noticed this, I guess this shouldn't have been deleted, could you please bring it back, so I can merge the PR?